app/create_file.py

```diff
@@ -1 +1,44 @@
-# write your code here
+import sys
+import os
+
+from datetime import datetime
+
+
+def create_file(name_file: str) -> None:
+    flag = "a" if os.path.exists(name_file) else "w"
+    with open(name_file, flag) as file:
+        first_line = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        if flag != "w":
+            first_line = "\n" + first_line
+        file.write(first_line)
+        line = ""
+        while line != "stop":
+            line = input("Enter content line: ")
+            if line != "stop":
+                file.write("\n" + line)
+        file.write("\n")
+
+
+def create_dirs(path_dir: list) -> None:
+    path = os.path.join(*path_dir)
+    os.makedirs(path, exist_ok=True)
+    os.chdir(path)
+
+
+if __name__ == "__main__":
+    command = sys.argv
+    if "-d" and "-f" in command:
+        if command.index("-d") < command.index("-f"):
+            start_index = command.index("-d")
+            finish_index = command.index("-f")
+            create_dirs(command[start_index + 1:finish_index])
+            create_file(command[-1])
+        else:
+            start_index = command.index("-d")
+            create_dirs(command[start_index + 1:])
+            create_file(command[command.index("-f") + 1])
```

Comment:
```suggestion
       f_index = command.index("-f")
       d_index = command.index("-d")
        if d_index < f_index:
            create_dirs(command[d_index + 1:f_index])
            create_file(command[-1])
        else:
            create_dirs(command[d_index + 1:])
            create_file(command[f_index + 1])
```
No need so much times calculate f_index and d_index